pkg/artifactory/resource_artifactory_virtual_bower_repository.go

```diff
@@ -0,0 +1,67 @@
+package artifactory
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceArtifactoryBowerVirtualRepository() *schema.Resource {
+
+	const packageType = "bower"
+
+	var bowerVirtualSchema = mergeSchema(baseVirtualRepoSchema, map[string]*schema.Schema{
+		"external_dependencies_enabled": {
+			Type:        schema.TypeBool,
+			Default:     false,
+			Optional:    true,
+			Description: "(Optional) When set, external dependencies are rewritten. Default value is false.",
+		},
+		"external_dependencies_remote_repo": {
+			Type:             schema.TypeString,
+			Optional:         true,
+			ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotEmpty),
+			RequiredWith:     []string{"external_dependencies_enabled"},
+			Description:      "(Optional) The remote repository aggregated by this virtual repository in which the external dependency will be cached.",
+		},
+		"external_dependencies_patterns": {
+			Type:     schema.TypeList,
+			Optional: true,
+			ForceNew: true,
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
+			RequiredWith: []string{"external_dependencies_enabled"},
+			Description: "(Optional) An Allow List of Ant-style path expressions that specify where external dependencies may be downloaded from. " +
```

Comment:
You should try to validate this. There are packages out there

Reply:
Attempted to use "github.com/vibrantbyte/go-antpath/antpath" ... was not quick fix.
Will address this antpath validation as a separate PR. Thanks.